pootle/core/views.py

```diff
@@ -632,11 +634,21 @@ def get_context_data(self, *args, **kwargs):
         ctx, cookie_data = self.sidebar_announcements
         ctx.update(
             super(PootleBrowseView, self).get_context_data(*args, **kwargs))
+
+        project_code = self.project.code if self.project else None
```

Comment:
I see you mean the [`LanguageBrowseView`'s `language` property](https://github.com/translate/pootle/blob/db299a6cc1585c9b0ee4be634905cf93b9b57012/pootle/apps/pootle_language/views.py#L80-L84). I don't quite get that either, (and `LanguageMixin` already returns the object, by the way).